content/doc/book/pipeline/jenkinsfile.adoc

```
@@ -490,48 +419,28 @@ pipeline {
 }
 // Script //
 ----
-<1> You can reference the two credential environment variables (defined in this
-Pipeline's <<syntax#environment,`environment`>> directive), within this stage's
-steps using the syntax `$AWS_ACCESS_KEY_ID` and `$AWS_SECRET_ACCESS_KEY`. For
-example, here you can authenticate to AWS using the secret text credentials
-assigned to these credential variables. +
-To maintain the security and anonymity of these credentials, if the job
-displays the value of these credential variables from within the Pipeline (e.g.
-`echo $AWS_SECRET_ACCESS_KEY`), Jenkins only returns the value "`+****+`" to
-reduce the risk of secret information being disclosed to the console output and any
-logs. Any sensitive information in credential IDs themselves (such as usernames)
-are also returned as "`+****+`" in the Pipeline run's output. +
-This only reduces the risk of **accidental exposure**.  It does
-not prevent a malicious user from capturing the credential value
-by other means. A Pipeline that uses credentials can also disclose
-those credentials.  Don't allow untrusted Pipeline jobs to use trusted
-credentials.
-<2> In this Pipeline example, the credentials assigned to the two `AWS_...`
-environment variables are scoped globally for the entire Pipeline, so these
-credential variables could also be used in this stage's steps. If, however, the
-`environment` directive in this Pipeline were moved to a specific stage (as is
-the case in the <<#usernames-and-passwords,Usernames and passwords>> Pipeline
-example below), then these `AWS_...` environment variables would only be scoped
-to the steps in that stage.
+<1> You can reference the two credential environment variables (defined in this Pipeline's <<syntax#environment,`environment`>> directive), within this stage's steps using the syntax `$AWS_ACCESS_KEY_ID` and `$AWS_SECRET_ACCESS_KEY`.
+For example, here you can authenticate to AWS using the secret text credentials assigned to these credential variables.
+To maintain the security and anonymity of these credentials, if the job displays the value of these credential variables from within the Pipeline (e.g. `echo $AWS_SECRET_ACCESS_KEY`), Jenkins only returns the value "`+****+`" to reduce the risk of secret information being disclosed to the console output and any logs.
```

```suggestion
To maintain the security and anonymity of these credentials, if the job displays the value of these credential variables from within the Pipeline (such as `echo $AWS_SECRET_ACCESS_KEY`), Jenkins only returns the value "`+****+`" to reduce the risk of secret information being disclosed to the console output and any logs.
```
Suggestion to replace `e.g.` with `such as` per [microsoft style guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg)